AlbumDownloader.py

```diff
@@ -56,7 +59,8 @@ def find_album_songs(album_title, artist):
     if soup.find(attrs=GOOGLE_LENGTH_TAG_ATTRS) is not None:
         lengths = [tag.text for tag in soup.find_all(attrs=GOOGLE_LENGTH_TAG_ATTRS)]
     else:
-        lengths = [None] * len(songs)
+        url = findWikiUrl(album=album_title, artist=artist)
+        lengths = searchLengthInWiki(url=url,titles=songs)
```

Comment:
just put it in one line...
and remember what is asked about google search VS wiki search